my_project/tab_outdoor_comfort/app_outdoor_comfort.py

```diff
@@ -3,146 +3,186 @@
 import dash_bootstrap_components as dbc
 from my_project.global_scheme import (
     outdoor_dropdown_names,
+    tight_margins,
+    month_lst,
+    container_row_center_full,
     container_col_center_one_of_three,
 )
 from dash.dependencies import Input, Output, State
 from my_project.template_graphs import (
     heatmap_with_filter,
     thermal_stress_stacked_barchart,
 )
-from my_project.utils import generate_chart_name, generate_units_degree, generate_units
+from my_project.utils import (
+    title_with_tooltip,
+    generate_chart_name,
+    generate_units_degree,
+    generate_units,
+)
 import numpy as np
 from app import app
 
 
-def layout_outdoor_comfort():
-    return html.Div(
-        className="container-col",
+def inputs_outdoor_comfort():
+    return dbc.Row(
+        className="container-row full-width three-inputs-container",
         children=[
-            html.Div(
-                className="container-row full-width three-inputs-container",
+            dbc.Col(
+                md=6, sm=12, xs=12,
```

Comment:
No need to have xs I believe since once the size it is small then the column width will be set to 12 already.